### ISSUE TITLE
generate_events fails for the FORTRAN backend

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#690]
I am continuing to debug my MR #688 based on Stefan's #620.

For the cuda and cpp backend, generate_events is ok in tlau/lauX.sh. For the fortran backend it fails.
```
    INFO: Running Survey
    Creating Jobs
    Working on SubProcesses
    INFO:     P1_gg_ttx
    INFO: Building madevent in madevent_interface.py with 'FORTRAN' matrix elements
    INFO:  Idle: 1,  Running: 0,  Completed: 0 [ current time: 19h44 ]
    INFO:  Idle: 0,  Running: 0,  Completed: 1 [  0.041s  ]
    INFO:  Idle: 0,  Running: 0,  Completed: 1 [  0.041s  ]
    INFO: End survey
    refine 10000
    Creating Jobs
    INFO: Refine results to 10000
    INFO: Generating 10000.0 unweighted events.
    Error when reading /data/avalassi/GPU2023/madgraph4gpuX/epochX/cudacpp/gg_tt.mad/SubProcesses/P1_gg_ttx/G1/results.dat
    Command "generate_events -f" interrupted with error:
    FileNotFoundError : [Errno 2] No such file or directory: '/data/avalassi/GPU2023/madgraph4gpuX/epochX/cudacpp/gg_tt.mad/SubProcesses/P1_gg_ttx/G1/results.dat'
    Please report this bug on https://bugs.launchpad.net/mg5amcnlo
    More information is found in '/data/avalassi/GPU2023/madgraph4gpuX/epochX/cudacpp/gg_tt.mad/run_01_tag_1_debug.log'.
    Please attach this file to your report.
    quit
    INFO:
```

I need to debug this more than expected, so I open a ticket

**Comment by valassi** [comment]
I think I understood it - the 'bridge' mode must be set to 0 for fortran. I will change that

**Comment by valassi** [comment]
Essentially, the bridge_mode currently should be 1 in cudacpp and 0 in fortran. This means that there is an interplay between the value of the cudacpp_backend (formerly exec_mode) card and the fbridgemode card, which is an issue.

In particular, these parameters must currently be set in two places
- gen_ximprove.py, where they are read from the runcards (and so in principle one can tweak fbridge mode because the backend is also known)
- refine.sh... here actually @roiser told me that we should use env variables (the current version I took from his wip #620 is hardcoded to 1)

As a very quick workaround: I will change driver.f to accept bridge mode =1 also in fortran.

Eventually: as suggested by @oliviermattelaer I should actually remove these extra parameters from the input file (#658). I can add them as env variables.

**Comment by valassi** [comment]
This is fixed by MR #688 - wher ein the end I removed the two extra parameters (#658)
